citar-file.el

```diff
@@ -106,36 +115,42 @@ Example: ':/path/to/test.pdf:PDF'."
          (mapcar (apply-partially #'expand-file-name fn) dirs)))
      parts)))
 
-(defun citar-file--possible-names (key dirs extensions &optional entry)
+(defun citar-file--possible-names (key dirs extensions &optional entry find-additional)
```

Comment:
```suggestion
(defun citar-file--possible-names (key dirs extensions &optional entry)
```
I'm leaning towards just checking the `defcustom` inside this function to see whether to do the directory scan. Otherwise the function arguments list seems to get unwieldy for no real benefit.

Then you would just make the following change on line 468 in `citar.el` (in `citar--format-candidates`)
```elisp
       (let* ((citar-find-additional-files nil) ; add this line
              (files
               (when (citar-file--files-for-entry
```